tests/assets/deps-package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Well, there are no `devDeps` or `deps` here, so the test will not check if any dependencies were correctly updated. I think default test case should cover that?

Reply:
> When you run tests against empty repo it throws an error (but then proceeds anyway)

> From what I see the error is because there is no setup-workflows.yml file there. Why it checks it and reports this as an error? Since tests pushes this file I would assume it usually won't be there thinking

I mention that partially here: https://github.com/ckeditor/ckeditor4-workflows-common/pull/15#issuecomment-824042833

GH Client throws an exception for `404` (which in this case is not an error, but useful information). Because there is no file on the repo, we don't want to extract SHA from that old file. We need to make this request - even if the URL is invalid for a non-existing file.

Previously I decided to `try...catch` any error right in the `client.js` module - to not spread them on the entire application. Request results are still returned, but also logged at the screen just in case. A similar 404 error could mean that someone provides the wrong GH rest API URL. But you can always verify your results via status. I don't want to `try...catch` each call in the application. So this log could be removed. Verifying status seems to be less code mess than wrapping everything in `try...catch` :thinking: WDYT?